Habitica/src/main/java/com/habitrpg/android/habitica/helpers/FirstDayOfTheWeekHelper.java

```diff
@@ -8,34 +8,39 @@
 public class FirstDayOfTheWeekHelper {
 
     private int firstDayOfTheWeek;
+    private int dailyTaskFormOffset;
 
-    public FirstDayOfTheWeekHelper(String firstDayOfTheWeekSharedPref) {
+    private FirstDayOfTheWeekHelper(int dailyTaskFormOffset, int firstDayOfTheWeek) {
+        this.dailyTaskFormOffset = dailyTaskFormOffset;
+        this.firstDayOfTheWeek = firstDayOfTheWeek;
+    }
+
+    public static FirstDayOfTheWeekHelper newInstance(String firstDayOfTheWeekSharedPref) {
         switch (firstDayOfTheWeekSharedPref){
             case "1":
```

Comment:
According to [this](stackoverflow.com/questions/11346916), Android uses strings arrays for both entries and entry values and that's all that's supported. Nevertheless, I went ahead and parsed the parameters from SharedPreferences so I could change the case values to the integer constants resulting in better readability.